modules/vector-index/partials/create-hyperscale-index-capella-ui.adoc

```diff
@@ -0,0 +1,85 @@
+== Create a HyperScale Vector Index with the Capella UI
+
+To create a HyperScale Vector index with the Capella UI: 
+
+. On the *Operational Clusters* page, select the operational cluster where you want to create a HyperScale Vector index.
+. Go to menu:Data Tools[Vector Indexes].
+. Click btn:[Create Vector Index].
+. Choose btn:[HyperScale Vector Index].
+. In the *Index Name* field, enter a name for your new Vector index.
++
+[NOTE]
+==== 
+Your index name must start with an alphabetic character (a-z or A-Z). It can only contain alphanumeric characters (a-z, A-Z, or 0-9), hyphens (-), or underscores (_).
+
+Your index name must be unique inside your selected bucket and scope.
+You cannot have 2 indexes with the same name inside the same bucket and scope.
+====
+. In the *Bucket*, *Scope*, and *Collection* lists, select the bucket, scope, and collection where you have documents that contain a vector field.
+. Under *Vector Field*, in the *Select Field* list, select the vector field to use in your HyperScale Vector index.
+. (Optional) To add a field to use as a filter on the data included in your HyperScale Vector index, click btn:[+ Add Filter Field].
+.. Under *Filter Fields*, select the field you want to use to filter your documents.
+.. Under *Include in Index*, choose whether the previously selected field should be included in your index and returned in search results.
```

Comment:
Following discussion in the comments, my suggestion is just to simplify this as follows. 

```suggestion
.. Under *Include in Index*, choose whether the previously selected field should be included in your index.
```